TrainModel/src/trainmodel/view/TrainModelUiController.java

```diff
@@ -185,42 +191,66 @@ private void end_failure_mode() {
 
   @Override
   public void initialize(URL location, ResourceBundle resources) {
+    initializeTrainModel();
     initializeStatusLabels();
     initializeStatusIcons();
     initializeButtonHandlers();
   }
 
+  private void initializeTrainModel() {
+    trainModelController = (TrainController) TrainControllerFactory
+        .createTrainController("001", "RED");
+    trainModel = (TrainModel) TrainModelFactory
+        .createTrainModel(trainModelController,"001", "RED");
+  }
+
```

Comment:
Won't this make every id == 001?

Reply:
It will. I still have to change that later to get the train ID from the trainController I believe.

Reply:
@michaelplazek I added commits to this PR to address those issues you had. I added TODO: comments in locations where I need to add development. I will include those in another PR if that works.

Reply:
Actually. I think this method will get refactored or completely deleted once I fully integrate my module with the CTC and MainMenu. Because Ideally those hard coded values will be given by the CTC. Right?